### PR TITLE
show more/less toggle to appear on overview

### PR DESCRIPTION
**Pull request by flacoman91**
fixing toggle to appear on overview page

### CODE REVIEW

File: src/reducers/query.jsx [hunk 1]
```diff
@@ -998,6 +1009,8 @@ export default ( state = defaultQuery, action ) => {
     validateDateInterval( newState )
   }
 
+  // remove any filter keys with empty array
+  pruneEmptyFilters( newState )
```
Comment (JeffreyMFarley) [inline src/reducers/query.jsx:1013]: Wait... what happens in this sequence

1. Filter on `90210`
1. Filter on State = `Kansas`

Both filters get dropped?

Comment (flacoman91) [reply src/reducers/query.jsx:1017]: doesn't seem like its getting dropped from the query reducer
filter

Comment (JeffreyMFarley) [reply src/reducers/query.jsx:1017]:
```
    zip_code: [
      '90210'
    ],
    state: [
      'KS'
    ]
```

So the filter arrays have values, but the counts are zero.. gotcha

Comment (sephcoster) [reply src/reducers/query.jsx:1017]: Thanks for the clarification!

**Pull request merged**